docs/dagster-university/pages/dagster-essentials/extra-credit/asset-metadata-as-markdown.md

```diff
@@ -126,13 +126,13 @@ In Lesson 9, you created the `adhoc_request` asset. During materialization, the
    2. `base64.b64encode` encodes the image’s binary data (`image_data`) into base64 format.
    3. Next, the encoded image data is converted to a UTF-8 encoded string using the `decode` function.
    4. Next, a variable named `md_content` is created. The value of this variable is a Markdown-formatted string containing a JPEG image, where the base64 representation of the image is inserted.
-   5. Using `context.add_output_metadata`, the image is passed in as metadata. The metadata will have a `preview` label in the Dagster UI.
+   5. We are able to include the metadata on the asset by returning a `MaterializeResult` instance with the image passed in as metadata. The metadata will have a `preview` label in the Dagster UI.
```

Comment:
```suggestion
   5. To include the metadata on the asset, we returned a `MaterializeResult` instance with the image passed in as metadata. The metadata will have a `preview` label in the Dagster UI.
```

Kinda nitpicky (sorry!) but this small change makes the copy fit more with how the other bullets are currently structured.